apps/src/componentLibrary/tags/tags.module.scss

```diff
@@ -0,0 +1,113 @@
+@import "color.scss";
+@import "font.scss";
+@import "@cdo/apps/componentLibrary/typography/typography.module";
+
+// Tags common styles
+.tags {
+  display: flex;
+  column-gap: 0.5em;
+
+  > div {
+    display: flex;
+    align-items: baseline;
+    gap: 2px;
+    color: $neutral-dark;
+    padding: 2px 12px;
+    background-color: $brand_primary_light;
+    border-radius: 6.25rem;;
+    max-width: 15em;
+
+    span {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    i {
+      display: flex;
+      justify-content: center;
+    }
+
+    &:has(i:first-child) {
+      padding-inline-start: 8px;
```

Comment:
Out of curiosity, do we have a plan to create some tokens/constants for these and other padding/margin sizes?